code/src/components/AddTodo.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import Button from '@material-ui/core/Button'
+import TextField from '@material-ui/core/TextField'
+
+import { tasks } from '../reducers/tasks'
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+`
+const StyledButton = styled(Button)`
+  box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
+  padding: 7px 14px;
+  width: 100%;
+  height: 50px;
+  &:hover {
+    background-color: #5469d4;
+  }
+`
+const StyledTextField = styled(TextField)`
+  width: 100%;
+`
+
+export const AddTodo = () => {
+  const dispatch = useDispatch()
+  const [ text, setAddItem ] = useState('')
+  const [ id, setaddId ] = useState(0)
```

Comment:
I think you should stick to the conventions when it comes to naming state variables. In this case it would be `[text, setText] `and` [id, setId]`. Otherwise it becomes hard to see that e.g. `text `and `setAddItem `belong together.

Reply:
Done